api_tests/sanity/lnd.ts

```diff
@@ -0,0 +1,21 @@
+import { twoActorTest } from "../lib_sdk/actor_test";
+
+setTimeout(function() {
+    twoActorTest("sanity-lnd-alice-pays-bob", async function({ alice, bob }) {
+        await alice.startLnd();
+        await bob.startLnd();
+
+        await alice.fundLnd();
+
+        await alice.connectLnd(bob);
+        await alice.openChannel(bob);
+
+        const invoice = await bob.addInvoice(alice);
+        await alice.sendPayment(invoice);
```

Comment:
Are we testing that the channel setup was successful? Then this is ok. 

Alternatively you can test the proposed process from the spike, e.g.: 

```
const secret = alice.createSecret();
const invoice = bob.createHoldInvoice(hash(secret));
const invoice = bob.addHoldInvoice(invoice);
alice.payInvoice(invoice);
bob.settleInvoice(invoice);
```

Reply:
For me this goes to far into the proposed protocol.  This issue is just adding basic LND nodes and doing a _normal_ payment between two nodes.  I assume that asserting the channel setup successful is handled by the library code.

Reply:
Got it.